src/main/java/seedu/address/logic/InternshipLogicManager.java

```diff
@@ -62,6 +66,25 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
         return commandResult;
     }
 
+    public List<Task> getDueTasks() {
+        ObservableList<Task> allTasks = model.getInternshipData().getDueTaskList();
```

Comment:
how do you get the DueTaskList? currently they all return null

Reply:
Its created in the InternshipData.java
Hope I didnt misunderstand the qn